jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -56,6 +61,10 @@ public TaskSummary getTask(String serverTemplateId,
             return null;
         }
 
+        if (!specManagementService.getServerTemplate(serverTemplateId).getContainerSpec(containerId).getStatus().equals(KieContainerStatus.STARTED)) {
```

Comment:
Just thinking in terms of standards, should this be done in the UI so we can give the specific error cause?